File: Loops/palindrome.c
//Program to check a number is palindrome

#include <stdio.h>

int main(){
    int i,num = 126721,reverse = 0,original = num;
    while(num!=0){
        int digit = num%10;
        reverse = reverse*10 + digit;
        num = (num-digit)/10;
    }
    if(original == reverse){
        printf("Number is palindrome");
    }else{
        printf("Number is  not palindrome");
    }
}

File: Arrays/palindrome.js
//Program to check if the array is a palindrome (reads the same forward and backward).

#include <stdio.h>
int main(){
int arr[100],n,i;
    printf("Enter the limit:");
    scanf("%d",&n);
    printf("Enter the values:");
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    int isPalindrome = 0;
    for(i=0;i<n/2;i++){
        if(arr[i] == arr[n-i-1]){
            isPalindrome = 1;
            break;
        }
    }
    if(isPalindrome==1){
        printf("Array is palindrome");
    }else{
        printf("Not palindrome");
    }
}
